### PR TITLE
Multiple arguments `by` option

### PR DESCRIPTION
**Pull request by dmdashenkov**
#### Support for multiple-argument `by` annotations.

The syntax is:
```
MyType boundField = 42 [(by)= "qualifier.One.target_field|qualifier.Two.alternate_target_field"];
```
Wildcard syntax (`(by)="*.target_field|no_qualifier_field"`) is supported in cases when annotation `enrichment_for` gives a not ambiguous definition for the target type.

For FQN field names in the `enrichment_for` may be omitted (the same behavior as with a single-argument `by` annotation).

### CODE REVIEW

File: server/src/test/java/org/spine3/server/event/enrich/ReferenceValidatorShould.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    public void fail_if_no_mapping_function_is_defined() {
+        final EventEnricher mockEnricher = mock(EventEnricher.class);
+        when(mockEnricher.functionFor(any(Class.class), any(Class.class)))
+                .thenReturn(Optional.<EnrichmentFunction<?, ?>>absent());
+        final ReferenceValidator validator = new ReferenceValidator(mockEnricher,
+                                                                    UserDeletedEvent.class,
+                                                                    GranterEventsEnrichment.class);
+        validator.validate();
+    }
+
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/event/enrich/ReferenceValidatorShould.java]: Please remove an excessive line.

Comment (dmdashenkov) [inline server/src/test/java/org/spine3/server/event/enrich/ReferenceValidatorShould.java]: Done

**Comment by dmdashenkov** [comment]
@armiol PTAL again

**Comment by dmdashenkov** [comment]
@armiol Fixed issue with spaces in `by` argument.
PTAL again

**Comment by armiol** [comment]
@dmdashenkov LGTM.

**Pull request merged**